Mathlib/Data/Vector/Defs.lean

```diff
@@ -83,14 +138,35 @@ theorem cons_head_tail : ∀ v : Vector α (succ n), cons (head v) (tail v) = v
 def toList (v : Vector α n) : List α :=
   v.1
 
+@[simp] theorem ofVector_toList (v : _root_.Vector α n) : (ofVector v).toList = v.toArray.toList :=
```

Comment:
```suggestion
theorem ofVector_toList (v : _root_.Vector α n) : (ofVector v).toList = v.toArray.toList :=
```
I don't think either side is obviously better than the other here.

Reply:
Changed.